### PR TITLE
Changed base image and added necessary config

### PR DESCRIPTION
**Pull request by georak12**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,10 @@
-FROM rabbitmq:3.8.2-alpine
+FROM bitnami/rabbitmq:3.8
```
Comment (pinkavaj) [inline Dockerfile]: Please use full version, so the build is reproducible.

Upgrading to latest version (sinc it is a minor upgrade) might make a sense .

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
 
-RUN wget https://github.com/rabbitmq/rabbitmq-message-timestamp/releases/download/v3.8.0/rabbitmq_message_timestamp-3.8.0.ez && \
-    mv rabbitmq_message_timestamp-3.8.0.ez /plugins/
+USER root
 
+RUN apt-get update -y \
+    && apt-get install wget -y \
+    && wget https://github.com/rabbitmq/rabbitmq-message-timestamp/releases/download/v3.8.0/rabbitmq_message_timestamp-3.8.0.ez \
+    && mv rabbitmq_message_timestamp-3.8.0.ez /plugins
```
Comment (pinkavaj) [inline Dockerfile:10]: Consider cleaning the apt cache to remove unnecessary content form the image.

Comment (pinkavaj) [reply Dockerfile:10]: Consider adding `&& rm -rf /var/lib/apt/lists/*`

Comment (georak12) [reply Dockerfile:10]: done

**Pull request merged**